doc/radosgw/admin.rst

```diff
@@ -224,7 +224,7 @@ also remove a key. Options include:
 - ``--key-type=<type>`` specifies the key type. The options are: s3, swift
 - ``--access-key=<key>`` manually specifies an S3 access key.
 - ``--secret-key=<key>`` manually specifies a S3 secret key or a Swift secret key.
-- ``--gen-access-key`` automatically generates a S3 key.
+- ``--gen-access-key`` automatically generates a S3 access key.
```

Comment:
`$ radosgw-admin --help` shows different text. I hope it is fine `generate random access key (for S3)`.